pkg/datapath/linux/config/config.go

```diff
@@ -793,6 +793,9 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	// Write Identity and ClusterID related macros.
 	cDefinesMap["CLUSTER_ID_MAX"] = fmt.Sprintf("%d", option.Config.MaxConnectedClusters)
 
+	// Write ENABLE_PACKET_IP_TRACING macro from EnablePacketIPTracing value
+	cDefinesMap["ENABLE_PACKET_IP_TRACING"] = strconv.Itoa(int(option.Config.EnableIPOptionTracing))
```

Comment:
I forgot that in my review: you will probably want to add coverage for this in complexity tests (`bpf/complexity-tests/`). There's quite a bit of parsing logic under this macro, so you probably want to ensure it's covered so that you don't hit a verifier error when you enable it.

Reply:
Good call. when I finish addressing the other comments in this PR, i'll get to that.